init-odkx-sync-endpoint.py

```diff
@@ -153,7 +153,7 @@ def run_docker_builds():
 def run_sync_endpoint_build():
     os.system("git clone https://github.com/odk-x/sync-endpoint ; \
                cd sync-endpoint ; \
-               mvn clean install -DskipTests")
+               mvn -pl sync-endpoint-war,sync-endpoint-swarm,sync-endpoint-common-dependencies clean install -DskipTests")
```

Comment:
On my installation, it seems the project reference for sync-endpoint-swarm should be changed to "org.opendatakit:sync-endpoint-docker-swarm".
I.e. so that the build command becomes:
```code
mvn -pl sync-endpoint-war,org.opendatakit:sync-endpoint-docker-swarm,sync-endpoint-common-dependencies clean install -DskipTests
```

Reply:
you do not need the org.opendatakit: part. You are correct it should have been sync-endpoint-docker-swarm instead of sync-endpoint-swarm

Reply:
I first tried just to use "sync-endpoint-docker-swarm", but it refuses to work without the "org.opendatakit:" prefix?